lib/linter.js

```diff
@@ -281,6 +279,13 @@ class Linter {
     return fileResults;
   }
 
+  async getConfig() {
```

Comment:
Config should now be accessed through `getConfig` to ensure it's loaded instead of directly accessing the `config` member variable.